extensions/crc/src/extension.ts

```diff
@@ -232,39 +233,25 @@ function convertToProviderStatus(crcStatus: string): extensionApi.ProviderStatus
 
 async function startStatusUpdateTimer(): Promise<void> {
   statusFetchTimer = setInterval(async () => {
-    crcStatus = await commander.status();
-  }, 2000);
-}
-
-// async function crcBinary(): Promise<string> {
-//   return which('crc');
-// }
-
-function execPromise(command, args?: string[]): Promise<string> {
-  const env = process.env;
-  // In production mode, applications don't have access to the 'user' path like brew
-  return new Promise((resolve, reject) => {
-    let output = '';
-    const process = childProcess.spawn(command, args, { env });
-    process.stdout.setEncoding('utf8');
-    process.stdout.on('data', data => {
-      output += data;
-    });
-
-    process.on('close', () => resolve(output.trim()));
-    process.on('error', err => reject(err));
-  });
+    try {
+      crcStatus = await commander.status();
+    } catch (e) {
+      console.error('CRC Status tick: ' + e);
+      const unknownStatus: Status = { CrcStatus: 'Stopped', Preset: 'Unknown' };
+      crcStatus = unknownStatus;
+    }
+  }, 1000);
```

Comment:
Should we have this back to `2000` or is `1000` better for detection?

Reply:
I think it is better for state track